src/hud.h

```diff
@@ -48,6 +48,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define HUD_FLAG_MINIMAP_VISIBLE       (1 << 5)
 #define HUD_FLAG_MINIMAP_RADAR_VISIBLE (1 << 6)
 #define HUD_FLAG_BASIC_DEBUG           (1 << 7)
+#define HUD_FLAG_CHAT_VISIBLE          (1 << 8)
```

Comment:
not `HUD_FLAG_CHAT`?

Reply:
The rest use visible, keeping it consistent.